app/scripts/controllers/journalUpload.js

```diff
@@ -87,6 +90,12 @@ angular
       if (newReportId !== oldReportId) {
         $window.scrollTo(0, 0);
       }
+
+      // Prevent refresh on initial page load
+      if (!newReportId && !oldReportId) {
+        return;
+      }
```

Comment:
A similar thing here on page load is the only time that the ```oldReportId``` and ```newReportId``` should both be null. This will prevent an additional API call.